src/services/scheduler.ts

```diff
@@ -10,6 +10,8 @@ import { IasqlDatabase } from '../metadata/entity';
 import config from '../config';
 
 const workerShutdownEmitter = new EventEmitter();
+// no max number of listeners warning
+workerShutdownEmitter.setMaxListeners(0);
```

Comment:
I was to lazy to disconnect old dbs locally and I actually had more than 10 which is the default

Reply:
oof. So... I really want a stats dashboard with this reveal because I'm worried what's gonna happen to our production system if we explode the number of workers there...

Reply:
Not blocking, but we need a task for that, I think. But I wonder if Postgres can report # of connections to us on its own?

Reply:
I'm assuming the number of connections to postgres would be a much bigger problem. we at least know exactly how many dbs we have from the metadata repo. https://github.com/iasql/iasql-engine/issues/583